app/config.js

```diff
@@ -272,11 +272,13 @@ settings.downloadFormatParamsPhysical = {
 
 // Korp backend URL
 // Always use the backend on the production server
-var korpBackendServer = "https://www.kielipankki.fi";
-settings.korpBackendURL = korpBackendServer + "/korp/api8";
+// var korpBackendServer = "https://www.kielipankki.fi";
```

Comment:
I only now realized that the variable `korpBackendServer` is used also in `settings.downloadCgiScript` below (lines 281–282), so you should either not comment out line 275 or adjust the value of `settings.downloadCgiScript`. I’m sorry I only now noticed this.

Reply:
Right. Well, it was really my job to notice it :) I made it a more simply named variable and put it everywhere https://kielipankki.fi is neeeded.